models/web/snowplow_media_media_stats.sql

```diff
@@ -0,0 +1,266 @@
+{{
+config(
+materialized= 'incremental',
```

Comment:
Is there a reason why we're using dbt incrementalisation here rather than the custom snowplow incrementalisation?